source/FreeRTOS_TCP_Reception.c

```diff
@@ -331,14 +331,14 @@
                          */
                         if( pucPtr[ 0U ] == tcpTCP_OPT_SACK_A )
                         {
-                            ucLen -= 2U;
+                            ucLen -= 2;
```

Comment:
I am unsure about this change. `ucLen` is unsigned. Then while assigning 2 to it, why not make 2 explicitly signed by using `U`?

Reply:
This will also give MISRA violation.